### PR TITLE
[Feature]: GAのドキュメントを公開する

### LINKED ISSUE #21: [Feature]: GAのドキュメントを公開する
### Feature description

GAの箇所のREADMEを公開する．

特に，

1. 何をやっているのか
2. どうやって実行するのか
3. ~~実行における注意点~~

などを中心にまとめておく．必要に応じて英語版も作成しておく

### Motivation

ドキュメント，必要なので...

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Okabe-Junya**
close #21 

## WHAT

`/ga` のREADMEを書いた．日本語版，英語版の両方を書いた．

## OTHER INFO

更新を行う際は，日本語 <-->英語で対応が取れているのか確認する．あるいは自動生成させるシステムを作る．

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
@ciaran-regan-ie 

Can I ask for a review of the English version? (README.en.md) I think this README will help you to understand the experiments I have done.

Please point out any errors in grammar or writing style.

File: ga/README.en.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Contains a set of source code to perform parameter search of the urn-model using the Genetic Algorithm (GA).
+
+## What is actually done
+
+### Flow
+
+Parameter search by genetic algorithm is carried out in the following steps:
+
+1. randomly generate a population with a vector representing the parameters: `(rho, nu, friendship, recentness)` for each individual. 2.
```
Comment (ciaran-regan-ie) [inline ga/README.en.md:15]: Extra 2 here!

File: ga/README.en.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+2. run the pot model based on the parameters, calculate the goodness of fit from the results and record this
```
Comment (ciaran-regan-ie) [inline ga/README.en.md:16]: pot should say urn

File: ga/README.en.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+3. perform an evolutionary calculation of the population based on the goodness of fit
+    a. Select individuals with high goodness of fit and perform crossover
+    b. Perform mutations with a certain probability
+    c. Generate the next generation of the population.
+4. return to 2. However, the process ends after a certain number of generations.
+
+## Calculating the goodness of fit
+
+In particular, the goodness of fit required in 2 (for each individual) is specifically calculated as follows: 1.
```
Comment (ciaran-regan-ie) [inline ga/README.en.md:25]: Extra 1

File: ga/README.en.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. run the urn-model based on the parameters (`rho`, `nu`, `friendship`, `recentness`)
+2. record the "history of interactions" generated by the execution of the urn-model
+3. compute 10 metrics that characterise the network using the history recorded in 2. 4.
```
Comment (ciaran-regan-ie) [inline ga/README.en.md:29]: Extra 4

File: ga/README.en.md (hunk 2 above)
Comment (ciaran-regan-ie) [reply ga/README.en.md:16]: I think

**Review by ciaran-regan-ie — CHANGES_REQUESTED** [review]
A few small things! Looks great otherwise!

File: ga/README.en.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# Parameter search by Genetic Algorithm
+
+[日本語](README.md) | English
+
+## Overview
+
+Contains a set of source code to perform parameter search of the urn-model using the Genetic Algorithm (GA).
```
Comment (ciaran-regan-ie) [inline ga/README.en.md:7]: Small thing but I think this sounds more natural:

Contains the source code to perform a parameter search of the urn-model using the Genetic Algorithm (GA).

File: ga/README.en.md (hunk 2 above)
Comment (Okabe-Junya) [reply ga/README.en.md]: resolved https://github.com/tsukuba-websci/MPS-TOM-Urnmodel/pull/26/commits/7cd7f15961ab9c51c16eb4ad06c610bbd7b54b53

**Comment by Okabe-Junya** [comment]
@ciaran-regan-ie 

Thank you for your review!! && I resolved!!

deep-kun automatically made extra <N>...

**Review by ciaran-regan-ie — APPROVED** [review]

**Pull request merged**